test/unit/lbp-wrapper-factory.js

```diff
@@ -69,89 +66,94 @@ describe(">> Contract: WrapperFactory", () => {
       // Need to solve this in tokens.js helper file for > 2 tokens.
       tokenAddresses = sortedTokens.map((token) => token.address);
     });
-    it("$ deploy WrapperFactory", async () => {
-      setup.wrapperFactory = await init.getContractInstance(
-        "WrapperFactory",
+    it("$ deploy LBPWrapperFactory", async () => {
+      setup.LbpWrapperFactory = await init.getContractInstance(
+        "LBPWrapperFactory",
         owner,
-        [setup.lbpFactory.address, SWAP_FEE_PERCENTAGE]
+        [setup.lbpFactory.address]
       );
-      expect(await setup.wrapperFactory.LBPFactory()).to.equal(
+      expect(await setup.LbpWrapperFactory.LBPFactory()).to.equal(
         setup.lbpFactory.address
       );
-      expect(await setup.wrapperFactory.swapFeePercentage()).to.equal(
-        SWAP_FEE_PERCENTAGE
-      );
     });
   });
   context("» set MasterCopy of LBPWrapper", () => {
     it("$ reverts on zero address", async () => {
       await expectRevert(
```

Comment:
can we use same way of testing, below we have used `expect().to.be.revertedWith()`, let stick to one.

Reply:
I'm changing it but this is actually something you have written :D

Reply:
hm, I don't remember tbh :D